homeassistant/components/doorbird.py

```diff
@@ -13,7 +13,7 @@
 from homeassistant.components.http import HomeAssistantView
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['DoorBirdPy==0.1.2']
+REQUIREMENTS = ['DoorBirdPy==0.1.3']
```

Comment:
You need to run `script/gen_requirements_all.py` to also update `requirements_all.txt`

Reply:
Sorry. I'm closing this, fixing it and reopening.